packages/runtime/container-runtime/src/summarizerClientElection.ts

```diff
@@ -144,5 +149,5 @@ export class SummarizerClientElection
     }
 
     public static readonly clientDetailsPermitElection = (details: IClientDetails): boolean =>
-        details.capabilities.interactive && details.type !== summarizerClientType;
+        details.capabilities.interactive || details.type === summarizerClientType;
```

Comment:
Doesn't this essentially mean all clients?

Reply:
I'm not sure. It might mean that today, though you could imagine other kinds of non-interactive clients besides summarizers.

Reply:
Out of curiosity, as I'm trying to learn. Wouldn't it mean that you could have the same client being summarizer again, if he is the oldest one ?

Reply:
Makes sense! Thanks!